### PR TITLE
Update video

### PR DESCRIPTION
**Pull request by sunete**
support insert bilibili video

This is an enhancement or feature.

## Summary

Support insert bilibili video in the posts.

## How to insert bilibili video?
Replace `videoID` with the bvid of the video you want to post.

```
{% include video id="videoID" provider="bilibili" %}
```

### CODE REVIEW

File: _includes/video [hunk 1]
```diff
@@ -9,5 +9,9 @@
   <iframe src="https://www.youtube-nocookie.com/embed/{{ video_id }}" frameborder="0" allowfullscreen></iframe>
 {% elsif video_provider == "google-drive" %}
   <iframe src="https://drive.google.com/file/d/{{ video_id }}/preview" frameborder="0" allowfullscreen></iframe>
+{% elsif video_provider == "bilibili" %}
+<div class="responsive-video-container">
```
Comment (iBug) [inline _includes/video:13]: There's already a `<div class="responsive-video-container">` surrounding the whole `if` statement. Might be better to drop this duplicated `<div>` wrapper,

Comment (sunete) [reply _includes/video]: Yes. You're right. I have deleted the  duplicated `<div>` wrapper.

File: _includes/video [hunk 2]
```diff
@@ -9,5 +9,7 @@
   <iframe src="https://www.youtube-nocookie.com/embed/{{ video_id }}" frameborder="0" allowfullscreen></iframe>
 {% elsif video_provider == "google-drive" %}
   <iframe src="https://drive.google.com/file/d/{{ video_id }}/preview" frameborder="0" allowfullscreen></iframe>
+{% elsif video_provider == "bilibili" %}
+  <iframe src="//player.bilibili.com/player.html?bvid={{ video_id }}&page=1&as_wide=1&high_quality=1&danmaku=0" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
```
Comment (iBug) [inline _includes/video:13]: Another suggestion: Add `https:` before the URL so it stays in line with other items in the same list.

```diff
-  <iframe src="//player.bilibili.com/player.html?bvid={{ video_id }}&page=1&as_wide=1&high_quality=1&danmaku=0" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
+  <iframe src="https://player.bilibili.com/player.html?bvid={{ video_id }}&page=1&as_wide=1&high_quality=1&danmaku=0" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"></iframe>
```

Comment (sunete) [reply _includes/video]: :smile:Done.

**Pull request merged**